magda-registry-api/src/main/scala/au/csiro/data61/magda/registry/WebHookActor.scala

```diff
@@ -31,25 +105,38 @@ object WebHookActor {
           this.processAgain = true
         } else {
           this.isProcessing = true
-          println("WebHook Processing: STARTING")
-          processor.sendNotifications().map {
+
+          println(s"WebHook ${this.id} Processing: STARTING")
```

Comment:
Yep, good idea.  Though consider merging it first since the version in master does the same thing and this PR also fixes problems running connectors and things.